pkg/util/log/zap/zapcore_test.go

```diff
@@ -44,9 +45,23 @@ func parseShortFilePath(_ string) seelog.FormatterFunc {
 }
 
 func extractShortPathFromFullPath(fullPath string) string {
+	// Ask git for the root of the repository. This handles cases where
```

Comment:
Since this is purely a unit test, why not just return a fake path, eg. "my_file_path.go" ?
Or even simpler, we could remove the file path from the log format and expected strings

A unit test shouldn't require running `git` to pass 😄

Reply:
I like replacing the path with a placeholder instead, done.